test/MappingOperation/Implementations/MapToMultipleTest.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace AutoMapperPlus\MappingOperation\Implementations;
+
+use AutoMapperPlus\AutoMapper;
+use AutoMapperPlus\Configuration\AutoMapperConfigInterface;
+use AutoMapperPlus\Configuration\Options;
+use AutoMapperPlus\Exception\UnregisteredMappingException;
+use AutoMapperPlus\MappingOperation\ContextAwareOperation;
+use AutoMapperPlus\NameResolver\CallbackNameResolver;
+use AutoMapperPlus\Test\Models\Nested\ParentClass;
+use AutoMapperPlus\Test\Models\Nested\ParentClassDto;
+use AutoMapperPlus\Test\Models\Nested\PolymorphicChildA;
+use AutoMapperPlus\Test\Models\Nested\PolymorphicChildB;
+use AutoMapperPlus\Test\Models\Nested\PolymorphicDtoA;
+use AutoMapperPlus\Test\Models\Nested\PolymorphicDtoB;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class MapToTest
+ * @package AutoMapperPlus\MappingOperation\Implementations
+ * @group mappingOperations
+ */
+class MapToMultipleTest extends TestCase
+{
+    public function testItCanBeInstantiated()
+    {
+        $mapToAnyOf = new MapToMultiple([PolymorphicDtoA::class, PolymorphicDtoB::class]);
+
+        $this->assertEquals([PolymorphicDtoA::class, PolymorphicDtoB::class], $mapToAnyOf->getDestinationClassList());
+    }
+
+    public function testItCanMapToAClass()
+    {
+        $mapToAnyOf = new MapToMultiple([PolymorphicDtoA::class, PolymorphicDtoB::class]);
+        $mapToAnyOf->setOptions(Options::default());
+        $mapToAnyOf->setMapper(AutoMapper::initialize(function (AutoMapperConfigInterface $config) {
+            $config->registerMapping(PolymorphicChildA::class, PolymorphicDtoA::class);
+            $config->registerMapping(PolymorphicChildB::class, PolymorphicDtoB::class);
+        }
```

Comment:
`}));` on a single line looks cleaner imo - it might be another todo for me to add a linter to the codebase.